ir/src/main/scala/org/scalajs/core/ir/Infos.scala

```diff
@@ -47,6 +47,9 @@ object Infos {
       val methodsCalled: Map[String, List[String]],
       val methodsCalledStatically: Map[String, List[String]],
       val staticMethodsCalled: Map[String, List[String]],
+      /** For a Scala class, it is instantiated with a `New`; for a JS class,
+       *  its constructor is accessed with a `JSLoadConstructor`.
+       */
```

Comment:
I'm not sure this is the best place to put this... But I don't know a better place either... :-/